lib/main_common.pm

```diff
@@ -2415,6 +2415,9 @@ sub load_mitigation_tests {
     if (get_var('KVM_GUEST_MIGRATION')) {
         loadtest "cpu_bugs/kvm_guest_migration";
     }
+    if (get_var('ITLB')) {
```

Comment:
maybe it would be better to have single variable `CPU_BUGS=itlb,mitigations,spectre_v2,spectre_v2_user` ? and then 

```
    if (get_var('CPU_BUGS')) {
        <logic which split coma-separated string into list and load each item as file from cpu_bugs folder>
```

of course it is just idea for future improvements not must have in this PR